src/client/utils/API.tsx

```diff
@@ -0,0 +1,62 @@
+export {}
```

Comment:
Just calling this module `API` is too generic. Lots and lots of things are APIs. What is the API for? We should call it something like `userInfo.tsx` so we know what the purpose is.